### ISSUE TITLE
args[module] false-positive on selinux

### ISSUE THREAD

**Issue by rndmh3ro** [ansible/ansible-lint#2930]
##### Summary

The following code triggers ansible-lint.

```
- name: Configure selinux | selinux-01
  ansible.posix.selinux:
    policy: "{{ os_selinux_policy }}"
    state: "{{ os_selinux_state }}"
```

```
> ansible-lint roles/os_hardening/tasks/selinux.yml
WARNING  Listing 1 violation(s) that are fatal
args[module]: value of state must be one of: enforcing, permissive, disabled, got: {{ os_selinux_state }} (warning)
roles/os_hardening/tasks/selinux.yml:2 Task/Handler: Configure selinux | selinux-01

You can skip specific rules or tags by adding them to your configuration file:
# .config/ansible-lint.yml
warn_list:  # or 'skip_list' to silence them completely
  - experimental  # all rules tagged as experimental

                  Rule Violation Summary
 count tag          profile rule associated tags
     1 args[module]         syntax, experimental (warning)
```

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
ansible-lint 6.11.0 using ansible 2.14.1

```

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

```console (paste below)
# create a file selinux.yml with the following contents:

---
- name: Configure selinux | selinux-01
  ansible.posix.selinux:
    policy: "{{ os_selinux_policy }}"
    state: "{{ os_selinux_state }}"

```

Then lint it.

##### Desired Behavior

This code should not trigger a warning

##### Actual Behavior

```
> ansible-lint roles/os_hardening/tasks/selinux.yml
WARNING  Listing 1 violation(s) that are fatal
args[module]: value of state must be one of: enforcing, permissive, disabled, got: {{ os_selinux_state }} (warning)
roles/os_hardening/tasks/selinux.yml:2 Task/Handler: Configure selinux | selinux-01

You can skip specific rules or tags by adding them to your configuration file:
# .config/ansible-lint.yml
warn_list:  # or 'skip_list' to silence them completely
  - experimental  # all rules tagged as experimental

                  Rule Violation Summary
 count tag          profile rule associated tags
     1 args[module]         syntax, experimental (warning)
```

**Comment by apatard** [comment]
From a very quick look, this looks like a duplicate of https://github.com/ansible/ansible-lint/issues/2866 ?

**Comment by rndmh3ro** [comment]
> From a very quick look, this looks like a duplicate of #2866 ?

Yes, probably. I looked at various issues for this topic but did not see this one..
